docs/src/pages/Conformance.md

```diff
@@ -248,6 +248,13 @@ Reference searches on the IBM FHIR Server support search on:
 
 We recommend using logical reference where possible.
 
+Elements of type Reference may contain a versioned reference, such as `Patient/123/_history/2`. When performing chained, reverse chained (`_has`), `_include`, or `_revinclude` searches on versioned references, the following rules apply:
+
+* If a resource has a reference that is versioned, and a chained search is performed using the element containing the versioned reference, the search criteria will be evaluated against the current version of the referenced resource, regardless of the version specified. For example, if a `Condition` resource contains a reference of `Patient/123/_history/1` in its `subject` element, and the current version of the `Patient/123` resource is 2, and a search of `Condition?subject.name=Jane` is performed, the search criteria will be evaluated against the current version of the `Patient/123` resource rather than the specified version. This is because the IBM FHIR Server only stores search index information for the current versions of resources. In the case where a chained search does not act on the referenced version of a resource, the search results will contain an `OperationOutcome` with a warning that indicates the logical id of the resource and the element containing the versioned reference.
+* If a resource has a reference that is versioned, and a reverse chain search is performed using the element containing the versioned reference, then the referenced resource can only be returned as a match if the version specified is the referenced resource's current version. For example, if a `Condition` resource contains a reference of `Patient/123/_history/2` in its `subject` element, and the current version of the `Patient/123` resource is 2, and a search of `Patient?_has:Condition:patient:code=1234-5` is performed and the `Condition` resource meets the search criteria, then the `Patient/123` resource will be returned as a match since the version specified in the reference is the current version of the `Patient/123` resource. If the current version of the `Patient/123` resource was 3, then the resource would not be returned as a match in the search results. This is because the IBM FHIR Server will only return the current versions of `match` resources in search results.
```

Comment:
Similar recommendation on the example/presentation.  It's so very dense. 

However... this line... 

> If a resource has a reference that is versioned, and a reverse chain search is performed using the element containing the versioned reference, then the referenced resource can only be returned as a match if the version specified is the referenced resource's current version.

Could it be rewritten to move the final if before the then, and explain the possible outcomes. such as 

> If a resource has a reference that is versioned, and a reverse chain search is performed using the element containing the versioned reference, and  if the version specified is the referenced resource's current version, then the referenced resource can only be returned as a match

Reply:
I thought it was a little more understandable the way I have it. If you feel strongly I can change it.